src/frontend/src/pages/build/BuildDetail.tsx

```diff
@@ -542,8 +542,8 @@ export default function BuildDetail() {
             editAction={editBuild.open}
             editEnabled={user.hasChangePermission(ModelType.part)}
             imageUrl={build.part_detail?.image ?? build.part_detail?.thumbnail}
-            breadcrumbs={[
-              { name: t`Manufacturing`, url: '/manufacturing' },
+            breadcrumbs={[{ name: t`Manufacturing`, url: '/manufacturing' }]}
+            last_crumb={[
```

Comment:
It is a small request but can we make this `lastCrumb` instead of `last_crumb` - the convention here is for camelCase

Reply:
done